thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -487,6 +501,9 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
         unsigned totalActiveWriters;
         PointerArrayOf<CTarget> targets;
         std::atomic<bool> *sendersFinished = nullptr;
+        std::atomic<stat_type> numLocalRows {0};
```

Comment:
Minor: We normally use RelaxedAtomic for all stat values - which defaults the memory order for all operations to relaxed (and avoids having to specify it on each use).